dist/mapdc.css

```diff
@@ -425,6 +425,14 @@ body {
   padding: 0 4px;
   text-decoration: none; }
 
+.basemap-satellite .gradient-legend .block .text, .basemap-dark .gradient-legend .block .text, .basemap-odyssey .gradient-legend .block .text, .basemap-decimal .gradient-legend .block .text {
+  color: #eeeeee;
```

Comment:
Let's try to be consistent with values - either `#eee` here or `#000000` below. (I'd prefer `#eee` for repeated values, more concise and still precise.)

This is a stylelint rule we could enable if we ran that.

Also, `#000` for `black` below.

Reply:
@jrajav these aren't changes from this PR, they are from https://github.com/mapd/mapd-charting/pull/197/files so I feel that we should include them in a separate PR